docs/actions/adb_devices.md

```diff
@@ -39,11 +39,12 @@ end
 
 ## Parameters
 
-Key | Description
-----|------------
-  `adb_path` | The path to your `adb` binary
-
+Key | Description | Default
+----|-------------|--------
+  `adb_path` | The path to your `adb` binary | adb
 
+_- = this parameter doesn't have a default value_<br/>
```

Comment:
This `<br/>` could probably be replaced with `  ` (double space)

Reply:
It didn't work for some reason, and double spaces are way fragile than a `<br/>`, and can easily be removed by a trailing space removal like I have in Sublime Text. And it's also visible, and so clearer, than some invisible characters.